src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicMobsMobTag.java

```diff
@@ -468,9 +654,11 @@ else if (mechanism.matches("reset_target")) {
         // @tags
         // <MythicMobsMobTag.level>
         // -->
-        else if (mechanism.matches("level") && mechanism.requireInteger()) {
-            mob.setLevel(mechanism.getValue().asInt());
-        }
+        tagProcessor.registerMechanism("level", false, ElementTag.class, (object, mechanism, value) -> {
+            if (mechanism.requireDouble()) {
```

Comment:
Was previously an `int`, if MM changed to support decimals then that's fine, but should update the meta accordingly